lib/Sabberworm/CSS/Parsing/ParserState.php

```diff
@@ -212,19 +213,25 @@ public function consumeUntil($aEnd, $bIncludeEnd = false, $consumeEnd = false, a
 		$out = '';
 		$start = $this->iCurrentPosition;
 
-		while (($char = $this->consume(1)) !== '') {
-			if (in_array($char, $aEnd)) {
-				if ($bIncludeEnd) {
-					$out .= $char;
-				} elseif (!$consumeEnd) {
-					$this->iCurrentPosition -= $this->strlen($char);
+		try {
+			while (($char = $this->consume(1)) !== '') {
```

Comment:
Is there any situation where `consume(1)` would return the empty string? This condition seems like it’s left over from a previous refactoring…